File: 백준/레벨Bronze V/9498. 시험 성적/시험 성적.ts
import fs = require('fs');

const input = fs.readFileSync(0, 'utf-8').trim();
const score = parseInt(input);

let result: string;
if (score >= 90 && score <= 100) result = 'A';
else if (score >= 80) result = 'B';
else if (score >= 70) result = 'C';
else if (score >= 60) result = 'D';
else result = 'F';

console.log(result);

File: 백준/레벨Bronze V/9498. 시험 성적/시험 성적.py
score = int(input())
result = ""

if 90 <= score <= 100:
    result = "A"
elif 80 <= score:
    result = "B"
elif 70 <= score:
    result = "C"
elif 60 <= score:
    result = "D"
else:
    result = "F"

print(result)